<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="r_btn_closet">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>40</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 36, 106);
border:0px;</string>
   </property>
   <property name="text">
    <string>옷장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="r_btn_main">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 36, 106);
border:0px;</string>
   </property>
   <property name="text">
    <string>메인</string>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_year">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_day">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_state">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 13pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_img_clothes1">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>90</y>
     <width>361</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 64, 185);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes1_n">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>570</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>옷이 없어요</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_img_clothes2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>180</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_img_clothes3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>430</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes1_s">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>640</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes1_t">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>640</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes2_t">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>290</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 15pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes2_n">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>200</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes2_s">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>290</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 15pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes3_t">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>540</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 15pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes3_s">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>540</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 15pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_clothes3_n">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>450</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="r_la_tem">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(45, 63, 185);
border:0px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>381</width>
     <height>641</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>420</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>r_la_clothes1_n</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>r_btn_closet</zorder>
  <zorder>r_btn_main</zorder>
  <zorder>r_la_year</zorder>
  <zorder>r_la_day</zorder>
  <zorder>r_la_state</zorder>
  <zorder>r_img_clothes1</zorder>
  <zorder>r_img_clothes2</zorder>
  <zorder>r_img_clothes3</zorder>
  <zorder>r_la_clothes1_s</zorder>
  <zorder>r_la_clothes1_t</zorder>
  <zorder>r_la_clothes2_t</zorder>
  <zorder>r_la_clothes2_n</zorder>
  <zorder>r_la_clothes2_s</zorder>
  <zorder>r_la_clothes3_t</zorder>
  <zorder>r_la_clothes3_s</zorder>
  <zorder>r_la_clothes3_n</zorder>
  <zorder>r_la_tem</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
